src/app/zones/components/ZoneIngressDetails.vue

```diff
@@ -54,16 +54,16 @@
         <template #body>
           <AccordionList :initially-open="0">
             <AccordionItem
-              v-for="(value, key) in subscriptionsReversed"
-              :key="key"
+              v-for="(subscription, index) in subscriptionsReversed"
+              :key="index"
             >
               <template #accordion-header>
-                <SubscriptionHeader :details="value" />
+                <SubscriptionHeader :subscription="subscription" />
               </template>
 
               <template #accordion-content>
                 <SubscriptionDetails
-                  :details="value"
+                  :subscription="subscription"
```

Comment:
nit:

Wondering why sometimes these are:

`:subscription="subscription"`, sometimes `:subscription="description"` and sometimes `:subscription="insight"`.

Might it be better to use something more generic like:

`:subscription="item"`

The `item` is then described by the name of the attribute, in this case a subscription